packages/hardhat-core/test/internal/hardhat-network/stack-traces/test.ts

```diff
@@ -247,6 +247,12 @@ function compareStackTraces(
   description: StackFrameDescription[],
   runs: number | undefined
 ) {
+  assert.equal(
+    trace.length,
+    description.length,
+    `Expected a trace of length ${description.length} but got one with length ${trace.length}`
```

Comment:
I think I removed this a long time ago because it was annoying when debugging. Things changed a lot, so if you think that's not the case anymore, fine for me.

Reply:
It was useful at some point in the refactor. I guess if it becomes annoying we can remove it again.